docker/deployed/staging/api/config.json

```diff
@@ -53,6 +53,32 @@
         "StuckInterval": "10m",
         "MinBlockDepth": 0
       }
+    },
+    {
```

Comment:
New staging chain: Optimism Goerli. It will run in parallel with Optimism Kovan until Optimism Kovan gets deprecated